Src/PerformanceCollector/NetCore/Implementation/QuickPulse/QuickPulseServiceClient.cs

```diff
@@ -0,0 +1,372 @@
+﻿namespace Microsoft.ApplicationInsights.Extensibility.PerfCounterCollector.Implementation.QuickPulse
```

Comment:
I know, but the code differs significantly due to NetFull using HttpWebRequest and NetCore using HttpClient, and after a careful consideration I chose to split them as opposed to doing precompile conditions to avoid unnecessary risk of breaking anything for existing implementation.